Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -67,6 +67,10 @@
 #include <wtf/cocoa/RuntimeApplicationChecksCocoa.h>
 #endif
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
```

Comment:
I'm concerned about potential regressions when the feature is disabled so that is why I gated it

Reply:
I think this particular change is non controversial without the flag. If there are regressions, we should fix them, because we will eventually do this anyway for the new fullscreen API

Reply:
oh but we still need to gate on `#if ENABLE(FULLSCREEN_API)` because FullscreenManager.h only exists on iOS